src/ui/components/touchableLink/InternalTouchableLink.tsx

```diff
@@ -0,0 +1,29 @@
+import { useLinkProps, useNavigation } from '@react-navigation/native'
+import React, { FunctionComponent, useCallback } from 'react'
+
+import { pushFromRef, navigateFromRef } from 'features/navigation/navigationRef'
+import { UseNavigationType } from 'features/navigation/RootNavigator/types'
+import { TouchableLink } from 'ui/components/touchableLink/TouchableLink'
+import { InternalTouchableLinkProps } from 'ui/components/touchableLink/types'
+
+export const InternalTouchableLink: FunctionComponent<InternalTouchableLinkProps> = ({
+  navigateTo,
+  enableNavigate = true,
+  ...rest
+}) => {
+  const internalLinkProps = useLinkProps({ to: navigateTo ?? '' })
```

Comment:
Vu que `navigateTo` est toujours défini dans `InternalTouchableLink` :
```suggestion
  const internalLinkProps = useLinkProps({ to: navigateTo })
```